### PR TITLE
Copy edit 4.62.0 changelog

### PR DESCRIPTION
**Pull request by nmelehan**

### CODE REVIEW

File: src/content/changelog/4-62-0-2020-04-10.md [hunk 1]
```diff
@@ -16,14 +16,16 @@ changelog:
 
 - The View Kubernetes API Endpoint ([/lke/clusters/{clusterId}/api-endpoint](/api/v4/lke-clusters-cluster-id-api-endpoint)) endpoint is now deprecated in favor of List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)).
 
-- The `version` property of the LKE collection has been updated to be named `k8s_version`. This change affects the following endpoints:
+- The `version` property of an LKE cluster has been updated to be named `k8s_version`. This change affects the following endpoints:
```
Comment (nmelehan) [inline src/content/changelog/4-62-0-2020-04-10.md:19]: Wanted to clarify this a bit, as the LKE collection also has two endpoints for viewing/listing kubernetes versions, and those are unaffected.

File: src/content/changelog/4-62-0-2020-04-10.md [hunk 2]
```diff
@@ -16,14 +16,16 @@ changelog:
...
 
     - List Kubernetes Clusters ([GET /lke/clusters](/api/v4/lke-clusters))
     - Create Kubernetes Clusters ([POST /lke/clusters](/api/v4/lke-clusters/#post))
     - View Kubernetes Cluster ([GET /lke/clusters/{clusterId}](/api/v4/lke-clusters-cluster-id))
 
 ### Fixed
 
-- The [LKE collection's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
+- An [LKE cluster's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
```
Comment (nmelehan) [inline src/content/changelog/4-62-0-2020-04-10.md:27]: Just making this a little more specific that those properties belong to the cluster object

File: src/content/changelog/4-62-0-2020-04-10.md [hunk 3]
```diff
@@ -16,14 +16,16 @@ changelog:
...
 
-- The `nodes` array's `id` property for [Create Node Pool](/api/v4/lke-clusters-cluster-id-pools/#post), [View Node Pool](/api/v4/lke-clusters-cluster-id-pools-pool-id), and [Update Node Pool](/api/v4/lke-clusters-cluster-id-pools-pool-id/#put) was incorrectly documented as being of type `number`. This has been corrected and the property is now documented as being of type `string`.
\ No newline at end of file
+- The `id` property of a Node in an LKE Node Pool was incorrectly documented as being of type `number`. This has been corrected and the property is now documented as being of type `string`.
```
Comment (nmelehan) [inline src/content/changelog/4-62-0-2020-04-10.md:29]: Splitting first sentence across two sentences

**Review by leslitagordita — APPROVED** [review]
These are great. Thank you!

**Pull request merged**